### ISSUE TITLE
Bug: Different Key commands behave different to blur

### ISSUE THREAD

**Issue by signalwerk** [facebook/lexical#2868]
I just did my first steps lexical (0.3.11) coming from [slate](https://www.slatejs.org/). I still have to learn a lot to reproduce my current editor, but what I see so far is super nice I have to say 🤩.
One thing I think is probably not obvious for a new person is the handling of registered commands. I would consider it as a bug.

When I register a `blur()` on escape-key everything works like a charm and as expected.

```js
    editor.registerCommand(
      KEY_ESCAPE_COMMAND,
      () => {
        editor.blur();
        return true;
      },
      COMMAND_PRIORITY_EDITOR
    );
```

but when I register a `blur()` on enter-key I don't get the blur.

```js
    editor.registerCommand(
      KEY_ENTER_COMMAND,
      () => {
        editor.blur();
        return true;
      },
      COMMAND_PRIORITY_EDITOR
    );
```

My intention is at the end to have a one-line input-field with some additional functionality. So, I would like to have on the enter-key the blur, and later I would also clean up the text entered by copy/past.

If there is a good reason for this behavior, please apologize for the buzz.

**Comment by trueadm** [comment]
In the enter key command handler, what if you also do `event.preventDefault()`?

**Comment by signalwerk** [comment]
@trueadm I also tried `event.preventDefault()` but was not successful. 

@yatagasaru that was totally it! thanks a lot with

```js
    editor.registerCommand(
      KEY_ENTER_COMMAND,
      (event) => {
        event.preventDefault();
        editor.blur();
        return true;
      },
      COMMAND_PRIORITY_LOW
    );
```

I was able to achieve the required behaviour. I returned for now `true` becaue I would like to have it «handled» and finish the dispatch for this command.

Thank you two! This was very helpful and super quick! 


(ps: I added on #2845 a note to improve the documentation for the `priority`. It was clearly me not understanding what the priority-levels mean. If you have a link or something for me I can also add it myself to the documentation and send a PR)
